content/resources/csc148.md

```diff
@@ -0,0 +1,76 @@
+---
+title: CSC148
+icon: /icons/csc148.svg
+desc: Intro to Computer Science
+link: csc148
+tags:
+  - utm-cs
+---
+
+# CSC148: Introduction to Computer Science
+
+#### Prerequisite: [CSC108](./csc108)
+
```

Comment:
We should also link the class website for 148 here using the 1x2 component

Reply:
don't we need to ask professors if we can do that?